test/utils_test_case.py

```diff
@@ -182,7 +182,13 @@ def assertNotEqual(self, x, y, prec=None, message=''):
             super(TestCaseBase, self).assertNotEqual(x, y, message)
 
 class TestCase(TestCaseBase):
-    def assertEqual(self, x, y, prec=None, message='', allow_inf=False):
+    def assertAlmostEqual(self, x, y, places=None, msg=None, delta=None, allow_inf=None, ignore_contiguity=False):
```

Comment:
is this function similar to torch.allclose, just that it's for nested tensors? if yes, it might be better to rename the function allclose or assertallclose

Reply:
its a wrapper layer around original TestCase.assertAlmostEqual. the wrapper is specifically for nested tesnors layer.